components/common/AccordianItemHeader/AccordianItemHeader.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
We should not use the phrase `All Rights Reserved` for the open source code. Please check the other places as well.

Reply:
Updated